projects/systelab-components/src/lib/datepicker/datepicker-time.component.html

```diff
@@ -19,14 +19,14 @@
 
     </ng-container>
 
-    <systelab-spinner class="d-flex h-100" id="hours" style="width: 80px;"
+    <systelab-spinner class="d-flex h-100" id="hours" style="width: 81px;"
```

Comment:
Could this width be a constant or variable in scss?

Reply:
You are right!!!!

Reply:
Done